packages/lumberaxe/.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+require:
+  - rubocop-powerhome
+
+AllCops:
+  TargetRubyVersion: 2.7
```

Comment:
@xjunior can/should we change this to support other versions?

Reply:
This should be the _oldest version of Ruby which your project supports_.